report/topology_test.go

```diff
@@ -231,12 +221,15 @@ func TestRenderByProcessPIDGrouped(t *testing.T) {
 			},
 		},
 		"apache": {
-			ID:          "apache",
-			LabelMajor:  "apache",
-			LabelMinor:  "",
-			Rank:        "215",
-			Pseudo:      false,
-			Adjacency:   NewIDList("curl", "localUnknown"),
+			ID:         "apache",
+			LabelMajor: "apache",
+			LabelMinor: "",
+			Rank:       "215",
+			Pseudo:     false,
+			Adjacency: NewIDList(
+				"curl",
+				"pseudo:;10.10.10.10;apache",
+				"pseudo:;10.10.10.11;apache"),
```

Comment:
We can leverage Go's tolerance of trailing commas to put the closing parens on a newline, making this much nicer.

```go
Adjacency: NewIDList(
	"curl",
	"pseudo:;10.10.10.10;apache",
	"pseudo:;10.10.10.11;apache",
),
```